extensions/Worker.Extensions.Abstractions/src/ExtensionInformationAttribute.cs

```diff
@@ -14,16 +14,24 @@ public class ExtensionInformationAttribute : Attribute
 
         public bool EnableImplicitRegistration { get; }
 
+        public bool SupportsBindingReferenceType { get; }
```

Comment:
Wonder if this shouldn't be named SupportsWorkerParameterBinding?

Reply:
With BindingData renamed to ModelBindingData, would `SupportsWorkerModelBinding` make sense?

Reply:
If we go with the defer bind naming, then this could be SupportsDeferredBinding